runtime/gc_vlhgc/CopyForwardCompactGroup.hpp

```diff
@@ -86,8 +93,6 @@ class MM_CopyForwardCompactGroup
 	void initialize(MM_EnvironmentVLHGC *env) {
 		_copyCache = NULL;
 		_copyCacheLock = NULL;
-		_TLHRemainderBase = NULL;
-		_TLHRemainderTop = NULL;
```

Comment:
a possible problem now, that these 2 pointers will have random values at first PGC?

Reply:
in MM_CopyForwardScheme::initialize(MM_EnvironmentVLHGC *env) line 318,  memset has been added for the protection.

```
	if (NULL == _compactGroupBlock) {
		return false;
	} else {
		memset(_compactGroupBlock, 0x0, allocateSize);
	}

```